client/post-editor/editor-title/index.jsx

```diff
@@ -4,6 +4,7 @@
 import React, { PropTypes } from 'react';
 import classNames from 'classnames';
 import omit from 'lodash/object/omit';
+import ReactDOM from 'react-dom';
```

Comment:
I have a feeling this will become an ongoing battle ( :smile: ), but per our naming conventions, this should be `ReactDom` ([reference](https://github.com/Automattic/wp-calypso/blob/master/docs/coding-guidelines/javascript.md#naming-conventions)).